services/cinema_hall.py

```diff
@@ -0,0 +1,14 @@
+from db.models import CinemaHall
+from django.db.models import QuerySet
+
+
+def get_cinema_halls() -> QuerySet:
```

Comment:
The return type of the `get_cinema_halls` function should not be `QuerySet` if it returns all objects. Instead, it should be `List[CinemaHall]` or similar, as it returns a list of objects, not a single object.